<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela_cadastro</class>
 <widget class="QMainWindow" name="janela_cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agenda | Novo Contato</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>561</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Novo Contato</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="bt_cadastrar">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>NOVO CONTATO</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_mensagem">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>59</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_mensagem_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>59</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#388E3C</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_emaill">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>47</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>E-mail:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_nome">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_telefone">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>60</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Telefone:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_email">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nome">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_telefone">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLista_ContatosC">
   <property name="text">
    <string>Contatos (Lista)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionNovo_Contato">
   <property name="text">
    <string>Novo Contato</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>le_nome</tabstop>
  <tabstop>le_telefone</tabstop>
  <tabstop>le_email</tabstop>
  <tabstop>bt_cadastrar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
